build/signedbuild.yml

```diff
@@ -11,335 +11,360 @@ variables:
+                                src\\AccessibilityInsights\\bin\\Release\\net48\\AccessibilityInsights.*.dll;\
+                                src\\AccessibilityInsights.VersionSwitcher\\bin\\Release\\net48\\*.exe;\
+                                src\\AccessibilityInsights.VersionSwitcher\\bin\\Release\\net48\\AccessibilityInsights.*.dll;\
+                                src\\Manifest\\bin\\Release\\net48\\AccessibilityInsights.*.dll;"
+
+        - task: securedevelopmentteam.vss-secure-development-tools.build-task-report.SdtReport@2
+          displayName: 'Create Security Analysis Report (BinSkim)'
+
+        - task: securedevelopmentteam.vss-secure-development-tools.build-task-publishsecurityanalysislogs.PublishSecurityAnalysisLogs@3
+          displayName: 'Publish Security Analysis Logs (BinSkim)'
+
+        - task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@2
+          displayName: 'Post Analysis (BinSkim)'
+
+        - task: VSTest@2
+          displayName: 'Test Assemblies **\release\*test*.dll;-:**\obj\**'
+          inputs:
+            testAssemblyVer2: |
+              **\*test*.dll
+              !**\obj\**
+            codeCoverageEnabled: false
+            platform: '$(BuildPlatform)'
+            configuration: release
+            rerunFailedTests: true
+            testFiltercriteria: 'TestCategory!=NoStrongName'
+
+        - task: PowerShell@2
+          displayName: 'Create tsa.config file (BinSkim)'
+          inputs:
+            targetType: "filePath"
+            filePath: tools\scripts\pipeline\build\create-tsa-config.ps1
+            arguments: '-InstanceUrl "$(TSAInstanceUrl)" -ProjectName "$(TSAProjectName)" -CodeBaseAdmins "$(TSACodeBaseAdmins)" -AreaPath "$(TSAAreaPath)" -IterationPath "$(TSAIterationPath)" -NotificationAliases "$(TSANotificationAlias)" -Tools "BinSkim" -OutputFile "$(Build.SourcesDirectory)\tsa.config"'
+
+        - task: securedevelopmentteam.vss-secure-development-tools.build-task-uploadtotsa.TSAUpload@2
+          displayName: 'TSA upload (BinSkim)'
+          condition: and(succeeded(), eq(variables.isMain, true))
+          inputs:
+            GdnPublishTsaOnboard: true
+            GdnPublishTsaConfigFile: '$(Build.SourcesDirectory)\tsa.config'
+
+        - task: CopyFiles@2
+          displayName: 'Copy Files for Signing Validation'
+          inputs:
+            SourceFolder: '$(Build.SourcesDirectory)\src'
+            Contents: |
+             AccessibilityInsights\bin\Release\net48\**\?(*.exe|*.dll)
+             AccessibilityInsights.VersionSwitcher\bin\Release\net48\**\?(*.exe|*.dll)
+             !AccessibilityInsights.VersionSwitcher\bin\Release\net48\**\?(Microsoft.Deployment.WindowsInstaller.dll|Newtonsoft.Json.dll)
+             MSI\bin\Release\msi\**\AccessibilityInsights.msi
+             Manifest\bin\Release\net48\AccessibilityInsights.Manifest.dll
```

Comment:
It looks like the indents of lines 365-368 are slightly off. Should we fix them  while we're touching this file or put it into a separate PR?

Reply:
corrected the indentation